packages/common/src/audius-query/types.ts

```diff
@@ -147,13 +156,50 @@ export type QueryHookOptions = {
   debounce?: number
 }
 
+export type PaginatedQueryHookOptions = QueryHookOptions & {
+  /**
+   * Page size to use.
+   * NOTE: if you change this value after the hook was initialized you will cause a cache miss and it will re-fetch data.
+   */
+  pageSize: number
+  /**
+   * Toggles single page data mode. If true the hook will only return the current page's data. Equivalent to usePaginatedQuery in the past
```

Comment:
maybe document contrasting behavior when it's false? maybe it's obvious, idk

Reply:
Hmm I feel like it should be somewhat obvious? I dont love the name though,